pubspec.yaml

```diff
@@ -15,6 +15,7 @@ dependencies:
   convert: ^2.0.1
   flutter_svg: ^0.15.0
   flare_flutter: ^1.5.4
+  meta: ^1.1.8
```

Comment:
why?

Reply:
This is the package responsible for the `@required` annotation.

Reply:
is it the one we normally use in flutter? because I don't remember to have to import a new package to use it in flutter regular app projects

Reply:
Yep, I did some research, and this package is also used by the flutter lint, I also thought at first that it was already bundled on flutter, but in the end I discovered that it isn't.